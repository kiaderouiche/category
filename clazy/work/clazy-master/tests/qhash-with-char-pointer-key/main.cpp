#include <QtCore/QHash>
#include <QtCore/QString>

void test()
{
    QHash<const char*, int> hash1; // Warn
    QHash<char, int> hash2;
    QHash<char**, int> hash3;
    QHash<int, int> hash4;
    QHash<char32_t, int> hash5;
    QHash<const char32_t*, int> hash6;
}
