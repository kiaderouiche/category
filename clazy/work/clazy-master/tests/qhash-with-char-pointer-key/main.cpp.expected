qhash-with-char-pointer-key/main.cpp:6:5: warning: Using QHash<const char *, T> is dangerous [-Wclazy-qhash-with-char-pointer-key]
